packages/packem/src/rollup/plugins/css/loaders/sass/utils/get-compile-function.ts

```
@@ -0,0 +1,82 @@
+let workQueue: PQueue | undefined;
+
+/**
+ * Verifies that the implementation and version of Sass is supported by this loader.
+ */
+const getCompileFunction = async (
+    implementation: typeof sass | typeof sassEmbedded | typeof nodeSass,
+    apiType: SassApiType,
+): Promise<
+    | ((sassOptions: nodeSass.SyncOptions) => nodeSass.Result)
+    // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
+    | ((sassOptions: nodeSass.SyncOptions) => Promise<void | nodeSass.Result>)
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)